src/main/java/de/hbz/lobid/helper/CreateWikidataNwbibMaps.java

```diff
@@ -9,18 +9,18 @@
 import java.nio.charset.StandardCharsets;
 
 import org.apache.commons.io.FileUtils;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.ResIterator;
+import org.apache.jena.rdf.model.Resource;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-import com.hp.hpl.jena.rdf.model.Model;
-import com.hp.hpl.jena.rdf.model.ModelFactory;
-import com.hp.hpl.jena.rdf.model.Property;
-import com.hp.hpl.jena.rdf.model.ResIterator;
-import com.hp.hpl.jena.rdf.model.Resource;
 
 /**
  * Creates nwbib maps used by morph.
- * 
```

Comment:
There seem to be a lot of these blankspace / formatting changes.

Reply:
Yes, they were formely unintendedly  introduced and now fixed by my IDE automatically.